toolkit/pallets/block-production-log/src/test.rs

```diff
@@ -9,12 +13,20 @@ fn make_id(i: u64) -> [u8; 32] {
 	id
 }
 
+fn current_block() -> BlockNumberFor<Test> {
+	<frame_system::Pallet<Test>>::block_number()
+}
+
 #[test]
 fn first_append_should_succeed() {
 	new_test_ext().execute_with(|| {
 		let call = Call::<Test>::append { block_producer_id: make_id(1) };
 		assert_ok!(call.dispatch_bypass_filter(RuntimeOrigin::none()));
 
+		assert_eq!(CurrentProducer::<Test>::get(), Some(make_id(1)));
+
+		Pallet::<Test>::on_finalize(current_block());
```

Comment:
we can also get block number from system pallet just in case, probably should give the same result as frame_system call
```suggestion
		Pallet::<Test>::on_finalize(System::block_number());
```

Reply:
yeah, that saves a couple of lines